### ISSUE TITLE
Address display type should be checksum not only lowercase

### ISSUE THREAD

**Issue by lxhyl** [ant-design/ant-design-web3#373]
Generally checksum is use to display Ethereum address in dApp. Like most popular dApp like UniSwap, Balancer, GMX...

There have a article to explain why Checksum is necessary.  

https://medium.com/coinmonks/ethereum-checksummed-addresses-importance-and-implementation-eef74aa3ae18

**Comment by yutingzhao1991** [comment]
good suggestion, how about add a new prop `checksum｀ for it?

when checksum is true, check address and show error when address is not correct.

**Comment by lxhyl** [comment]
Cool. And i'm happy to do this if you don't have the time

**Comment by yutingzhao1991** [comment]
> Cool. And i'm happy to do this if you don't have the time

Of course, you're welcome, but how to reflect the situation of an incorrect address in terms of style may require design support, which we can discuss further.

**Comment by yutingzhao1991** [comment]
Maybe create a new component AddressInput

**Comment by wangshangluobai** [comment]
Create a new **AddressInput**. Checksum is one of its default features (not controlled by props). When the input loses focus, perform address verification (standard wallet address verification to check if it is a checksum address, providing error prompts) and conversion (ultimately retaining the checksum address).

The component is based on the native HTML tag Input.

**Comment by yutingzhao1991** [comment]
```tsx
import { AddressInput, checksum } from '@ant-design/web3';

const { checksum } = AddressInput;

<AddressInput prefixCheck={} chain={Ethe...} />

<Form>
  <AddressInput />
</Form>
```
